source/content/guides/decoupled-sites/04-site.md

```diff
@@ -219,3 +217,43 @@ You are directed to a new page with the site name, deployment, and build informa
 
 </Tab>
 </TabList>
+
+## Stable URLs
+
+You can establish FQDN domains in decoupled environments using stable URLs.
+
+An environment relates to every code change made against the Git repository that triggers a build, in which that build generates an internet-ready site.
+
+Code change events that trigger a build include:
+
+* Push to branches
+* Opened pull requests 
+
+<Alert title="Note" type="info">
+
+NOTE: Pull requests from a Multidev branch that are made against the upstream, will trigger double builds.
+</Alert>
+
+### Types of Environments
+
+
+* **Production environment** 
+    * The production environment is the default branch of the repository
+    * This environment corresponds to the following stable URL pattern: `live-[site-name].appa.pantheon.site`
+
+* **Multidev environment (based on integration branches)**
+    * Multidev environments are based on branches belonging to the upstream repository. This branch will have the prefix `multi-` in the branch name.
```

Comment:
It might be worth nothing that the branch _must_ have the prefix of `multi-`, and not just that the branch _will_ have the prefix. Customers who create a branch without the `multi-` prefix could be annoyed that their branches didn't create builds (as I have firsthand experience of this).